src/components/index.ts

```diff
@@ -9,6 +9,7 @@ export * from './dds/Masthead';
 export * from './dds/Notification';
 export * from './dds/TextInput';
 export * from './dds/Typography';
+export * from './dds/Search';
```

Comment:
Minor thing, but for the sake of consistency - let's keep alphabetical order here:

```suggestion
export * from './dds/Notification';
export * from './dds/Search';
export * from './dds/TextInput';
export * from './dds/Typography';
```

Reply:
Sure 💪 

But in the future I wouldn't be worried about it, as it's not bringing any value from the perspective of components exposure
And having multiple components it may be time consuming to put every component in the right spot 😁

Reply:
Exports were adjusted within 7c7ef092bf7c46f8801318fbf46d086b57813109 🎉 
Funny part was that additional component was added, so I had to do a little investigation where to put `Search`, to match the alphabetical pattern 😄